### DISCUSSION TITLE
How I can change inner styles if the user hover on root element

### DISCUSSION THREAD

**Discussion by inthedark122** [master-co/css#100] [Help] [answered]
I want to change the inner block styles of any nested elements.

I made the examples that I want: https://codesandbox.io/s/change-inner-styles-on-root-block-pc02bz?file=/src/styles.css

If you hover over the grey block, you can see that colour on the nested `h2` tag changed.

How can I make this with `master-css`?

CSS example:

```
.block:hover > .inner {
  color: white;
}
```

**Comment by 1aron** [comment, marked as answer]
It's super easy:
```html
<div class="fg:white:hover>.inner">
    <div class="inner">
```
Or even use Master's reactive syntax:
```html
<div class="parent">
    <div class=".parent:hover_{fg:white}">
```
It must be admitted that the tutorial documentation of v1 is not friendly, we've made many improvements in v2, so stay tuned.

**Reply by inthedark122** [reply]
Thank you

It works perfectly
